export default async function () {
  try {
  } catch (error) {
    console.error('Default encountered an error:');
    console.error(error);
  }
}
